Sources/macros.js

```diff
@@ -972,6 +972,8 @@ export function newInstance(extend, className) {
     const model = {};
     const publicAPI = {};
     extend(publicAPI, model, initialValues);
+    const val = { ...initialValues, ...model };
```

Comment:
Are the `initialValues` suppose to override the `model` ones?

Reply:
Hasn't `model` been updated with `initialValues` in `extend()` ?